src/clr-angular/data/datagrid/datagrid-row.ts

```diff
@@ -129,9 +130,29 @@ export class ClrDatagridRow<T = any> implements AfterContentInit, AfterViewInit
     }
   }
 
+  /** By default every item is selectable */
+  @Input('clrDgSelectable')
+  public set clrDgSelectable(value: boolean) {
+    if ([SelectionType.Single, SelectionType.Multi].indexOf(this.selection.selectionType) > -1) {
+      this.selection.lockItem(this.item, !parseBooleanAttribute(value));
```

Comment:
Since we are only handling Angular inputs here, we don't need `parseBooleanAttribute()`, just set it to the value provided.

Also the selection type check is unnecessarily complex, just `this.selection.selectionType !== SelectionType.None` would suffice here.

```
if (this.selection.selectionType !== SelectionType.None) {
  this.selection.lockItem(this.item, value);
}
```